CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 2.1.1 (2023-08-24)
+--------------------------
+No code changes - this release addresses a vulnerability which is resolved by building a new docker image
```

Comment:
Here's an example of how the CHANGELOG entry was filled in a previous similar scenario: https://github.com/snowplow/stream-collector/blob/2.9.1/CHANGELOG#L27-L29

Reply:
I think [this](https://github.com/snowplow/snowplow-s3-loader/issues/265) was the first one of these we did.  It's nicer than the other example because the issue has a better description.

Reply:
Didn't realise there was an existing pattern. I've rebased with it done this way!